src/popup/router/index.ts

```diff
@@ -74,6 +74,8 @@ router.beforeEach(async (to, from, next) => {
     watchUntilTruthy(areAccountsRestored),
   ]);
 
+  await checkUserAuth();
```

Comment:
This change will result in this upon trying to read terms of use without initializing the wallet.
![image](https://github.com/user-attachments/assets/1b00adb3-2826-49b0-b237-6b62986d23f8)

Reply:
Nice catch. I fixed this by updating the `App.vue` `showRouter` logic.